lib/bot.ts

```diff
@@ -6,12 +6,38 @@ import fetch from 'node-fetch'
 const DEFAULT_REST_URL = 'https://eludris.tooty.xyz/'
 const DEFAULT_WS_URL = 'wss://eludris.tooty.xyz/ws/'
 
+/**
+ * The options for the bot.
+ */
 interface BotOptions {
+    /**
+     * The gateway URL to connect to.
+     * @public
```

Comment:
```suggestion
     * The gateway URL to connect to.
```
`@public` is for symbols that may be treated as private but should be documented as public. Reduces repetition.

Reply:
https://github.com/eludris-community/evangeline/pull/3/commits/808b143689c3e158c2ab576ae3b29eed54c3f0a8